src/Indice.Services/Extensions/IConfigurationExtensions.cs

```diff
@@ -76,6 +76,11 @@ public static class IConfigurationExtensions
     /// <remarks>Checks for the <strong>Proxy:Ip</strong> option in appsettings.json file.</remarks>
     public static string? GetProxyIp(this IConfiguration configuration) => configuration.GetSection(ProxyOptions.Name).GetValue<string>(nameof(ProxyOptions.Ip));
 
+    /// <summary>Gets the proxy's forward limit option.</summary>
```

Comment:
Always declare the configuration path in summary. Check the other extension methods for example.